presto-docs/src/main/sphinx/release/release-312.rst

```diff
@@ -0,0 +1,67 @@
+===========
+Release 312
+===========
+
+General Changes
+---------------
+
+* Fix incorrect results for queries using ``IS [NOT] DISTINCT FROM``. (:issue:`795`)
+* Fix ``array_distinct``, ``array_intersect`` semantics with respect to indeterminate
+  values (i.e., ``NULL`` or structural types containing ``NULL``). (:issue:`559`)
+* Fix failure when the largest negative ``BIGINT`` value (-9223372036854775808) is used as a
+  constant in a query. (:issue:`805`)
+* Improve reliability for network errors when using Kerberos with
+  :doc:`/security/internal-communication`. (:issue:`838`)
+* Improve performance of ``JOIN`` queries involving inline tables (``VALUES``). (:issue:`743`)
+* Improve performance of queries containing duplicate expressions. (:issue:`730`)
+* Improve performance of queries involving comparisons between values of different types. (:issue:`731`)
+* Improve performance of queries containing redundant ``ORDER BY`` clauses in subqueries. This may
+  affect the semantics of queries that incorrectly rely on implementation-specific behavior. The
+  old behavior can be restored via the ``skip_redundant_sort`` session property or the
+  ``optimizer.skip-redundant-sort`` configuration option. (:issue:`818`)
+* Improve performance of ``IN`` predicate containing a subquery. (:issue:`767`)
+* Improve support for corrected subqueries containing redundant ``LIMIT`` clauses. (:issue:`441`)
```

Comment:
I don't know what "corrected queries" means

Reply:
Is this supposed to be "correlated"?

Reply:
Oops, yeah, it's supposed to be correlated